doc/book/examples.md

```diff
@@ -171,6 +171,7 @@ class CacheListener implements ListenerAggregateInterface
     use ListenerAggregateTrait;
 
     private $cache;
+    private $listeners = [];
```

Comment:
@webimpress  Done, with another commit, do you know if I can revert commits on Github web without cloning and accessing CLI?